routes/orders.js

```diff
@@ -0,0 +1,47 @@
+const express = require("express");
+const ordersRouter = express.Router();
+const { requireUser, isAdmin } = require("./utils");
+
+ordersRouter.get("/orders", [requireUser, isAdmin], async (req, res, next) => {
+  try {
+    const orders = await getAllOrders();
+    res.send(orders);
+  } catch (error) {
+    next(error);
+  }
+});
+
+ordersRouter.get("/orders/cart", [requireUser, isAdmin], async (req, res, next) => {
+    try {
+      const orders = await getPendingOrdersByUsers();
```

Comment:
This should be sending back a single order that matches a userID for example... Read of the `req.user` to get this id